src/sqlfluff/api/simple.py

```diff
@@ -0,0 +1,54 @@
+"""The simple public API methods."""
+
+from ..core import Linter
+
+
+def _unfiy_str_or_file(sql):

```

Comment:
Typo in the function name: `_unfiy` -> `_unify`

Reply:
🤦